src/condor_starter.V6.1/docker_proc.cpp

```diff
@@ -271,7 +271,7 @@ DockerProc::LaunchContainer() {
 	TemporaryPrivSentry sentry(PRIV_USER);
 	std::string workingDir = starter->GetWorkingDir(0);
 	//std::string DockerOutputFile = workingDir + "/docker_stdout";
-	std::string DockerErrorFile  = workingDir + "/docker_stderror";
+	std::string DockerErrorFile  = workingDir + "/.docker_stderror";
```

Comment:
Should be being done with `std::filesystem::path`, but AFAIK Docker universe doesn't work on Windows, so this is OK for now.

The bigger issue here, and on all subsequent lines, is that this filename shows up in seven places but isn't a defined/manifest constant.